packages/webdriverio/src/commands/element/isClickable.js

```diff
@@ -0,0 +1,43 @@
+
+/**
+ *
+ * Return true if the selected DOM-element:
+ *
+ * - exists;
+ * - visible;
+ * - is within viewport (if not try scroll to it);
+ * - it's center is not overlapped with another element.
+ *
+ * otherwise return false.
+ *
+ * Please note that `isClickable` works only in web and webviews, it doesn't work in mobile app native context.
+ *
+ * <example>
```

Comment:
Can we add a `waitUntil` example with this command? I can see people would be really interested in that.

Reply:
Maybe just add waitForClickable command?

Reply:
Sounds good to me. This has been asked for a lot therefor I am fine with it

Reply:
added `waitUntil` example for now